tests/models/dsheetpiling/test_dsheetpiling_model.py

```diff
@@ -171,8 +170,22 @@ def test_given_parsed_input_when_serialize_then_same_content(self, filename: Pat
             if not (ds_key in output_keys):
                 errors.append(f"Key {ds_key} not serialized!")
                 continue
-            if not (ds_value == output_datastructure[ds_key]):
-                logging.warning(f"UNEQUAL: {ds_value} != {output_datastructure[ds_key]}")
+            od_value = output_datastructure[ds_key]
+            if not (ds_value == od_value):
+                logging.warning(f"UNEQUAL: {ds_value} != {od_value}")
+                try:
+                    # try getting better description of the problem
+                    od_dict = od_value.dict()
```

Comment:
nice

Reply:
Another suggestion for this test(test_import_output) is to run this for all the .shi files in the benchmark directory as an extra test

Reply:
I believe there's already a test for this.